vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1369,14 +1371,17 @@ def execute_model(
                     n + 1 - len(valid_sampled_token_ids[i]) if n > 0 else 0
                     for i, n in enumerate(num_draft_tokens)
                 ]
-                num_rejected_tokens = torch.tensor(
+                num_rejected_tokens_tensor = async_tensor_h2d(
                     num_rejected_tokens,
                     dtype=torch.int32,
-                    device=self.device,
-                )
+                    target_device=self.device,
+                    pin_memory=True)
+                num_tokens = spec_decode_metadata.total_num_scheduled_tokens - \
+                    sum(num_rejected_tokens)
```

Comment:
nit: please refrain from using `\`
```suggestion
                num_tokens = (spec_decode_metadata.total_num_scheduled_tokens - 
                    sum(num_rejected_tokens))
```